exchange/utils.go

```diff
@@ -46,14 +46,18 @@ func cleanOpenRTBRequests(ctx context.Context,
 	privacyConfig config.Privacy,
 	account *config.Account) (allowedBidderRequests []BidderRequest, privacyLabels metrics.PrivacyLabels, errs []error) {
 
-	impsByBidder, err := splitImps(req.BidRequest.Imp)
-	if err != nil {
-		errs = []error{err}
+	aliases, errs := parseAliases(req.BidRequest)
+	if len(errs) > 0 {
 		return
 	}
```

Comment:
I just moved it to the first statement of the  `cleanOpenRTBRequests` function, because aliases are needed in `buildBidderRequestsWithBidResponses`.  `buildBidderRequestsWithBidResponses` should be executed before everything else, because it modifies request - removes all imps with stored bid responses from imp array. 

```
        aliases, errs := parseAliases(req.BidRequest)
	if len(errs) > 0 {
		return
	}

	allowedBidderRequests = make([]BidderRequest, 0, 0)

	bidderToBidderResponse := buildBidderRequestsWithBidResponses(req, aliases)

       impsByBidder, err := splitImps(req.BidRequest.Imp)
...
```